frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -19,8 +49,142 @@ function AuthedView() {
+    setPerPage(perPage);
+  };
+
+  return (
+    <main>
+      <div className="md:my-10 my-5">
+        <h1 className="text-3xl  font-bold">Overview</h1>
+        <h4 className="text-sm mt-2 opacity-60">List of projects</h4>
+      </div>
+
+      <div className="flex my-3 flex-wrap items-center md:gap-3 gap-1">
+        <div className="flex md:my-5 md:w-[30%] w-[100%]  items-center">
+          <Search size={20} className="relative left-5" />
+          <Input
+            className="px-14 placeholder:text-gray-400  -mx-5 w-full my-1 text-sm focus-visible:right-0"
+            placeholder="Ex: ZaneOps"
+          />
+        </div>
+
+        <div className="md:w-fit w-full">
+          <Menubar className="border border-border md:w-fit w-full ">
+            <MenubarMenu>
+              <MenubarTrigger className="flex md:w-fit w-full ring-secondary md:justify-center justify-between text-sm items-center gap-1">
+                Status
+                <ChevronsUpDown className="w-4" />
+              </MenubarTrigger>
+              <MenubarContent className=" border min-w-[23rem] border-border md:min-w-6">
+                <MenubarContentItem icon={Rocket} text="Active" />
+                <MenubarContentItem icon={Trash} text="Archived" />
+              </MenubarContent>
+            </MenubarMenu>
+          </Menubar>
+        </div>
+      </div>
+
+      <Table>
+        <TableHeader className="bg-toggle">
+          <TableRow className="border-none">
+            <TableHead>Name</TableHead>
+            <TableHead className="flex items-center gap-2">
+              Last Updated
+              <ArrowDown size={15} />
+            </TableHead>
+            <TableHead>Status</TableHead>
+            <TableHead className="text-right">Actions</TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {projects.map((project) => (
+            <TableRow className="border-border cursor-pointer" key={project.id}>
+              <TableCell className="font-medium flex items-center gap-3">
+                <Folder size={18} />
+                {project.name}
+              </TableCell>
+              <TableCell>{project.updated_at}</TableCell>
+              <TableCell>
+                <StatusBadge color={getBadgeColor(project.tracker)}>
+                  {project.status}
+                </StatusBadge>
+              </TableCell>
+              <TableCell className="flex justify-end">
+                <div className="w-fit flex items-center gap-3">
+                  Settings
+                  <Settings width={18} />
+                </div>
+              </TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+
+      <div className="my-4">
+        <Pagination
+          totalPages={10}
+          currentPage={currentPage}
+          perPage={perPage}
+          onChangePage={handleChangePage}
+          onChangePerPage={handleChangePerPage}
```

Comment:
```suggestion
          onChangePage={() => setCurrentPage(page)}
          onChangePerPage={() => setPerPage(perPage)}
```

When the function is very simple, you can just inline it